modules/serverless-rn-1-5-0.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-5-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.5.0
+
+[id="new-features-1-5-0_{context}"]
+== New features
+* {ServerlessProductName} 1.5.0 is available on {product-title} 4.3 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.12.1.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.12.0.
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.12.1.
+* {ServerlessProductName} ingress implementation has been updated to use Kourier in place of Service Mesh. No user intervention is necessary, as this change is automatic when the {ServerlessOperatorName} is upgraded to 1.5.0.
+
+[id="fixed-issues-1-5-0_{context}"]
+== Fixed issues
+* In previous releases, {product-title} scale from zero latency caused a delay of approximately 10 seconds when creating pods. This issue has been fixed in the {product-title} 4.3.5 bug fix update.
+
+[id="known-issues-1-5-0_{context}"]
+== Known issues
```

Comment:
We would like to put https://issues.redhat.com/browse/SRVKS-447  as a known issue. @markusthoemmes do you agree?

Reply:
sure